addons/OPT/SHOP/fn_getprice.sqf

```diff
@@ -0,0 +1,59 @@
+/**
+* Description: Preis feststellung für Fahrzeug
+* 
+* 
+* Author:
+* [GNC]Lord-MDB
+*
+* Arguments:
+* 
+*
+* Return value:
+* 
+*
+* Server Only:
+* No
+* 
+* Global:
+* No
+* 
+* API:
+* No
+* 
+* Example:
+* 
+*/
+
+#include "macros.hpp"
+
+params [
+    ["_class", "", [""], [1]]
+];
+
+if (_class isEqualTo "") exitWith {0};
+
+private _unitCost = 0;
+private _index = (GVAR(all) apply {toLower (_x select 0)}) find (toLower _class);
```

Comment:
Für Fälle wie diesen wäre es performater, wenn in `GVAR(all)` ainfach schon alles nur in Kleinbuchstaben rein geschrieben würde. Dann könnte man sich hier die Erzeugung eines temporären Arrays sparen und einfach direkt auf `GVAR(all)` zugreifen.